data/application.css

```diff
@@ -25,38 +25,70 @@ textview.terminal selection {
     color: #252e32;
 }
 
-.ext2, .ext3, .ext4 {
-    background-color: rgb(114, 147, 203);
+.disk-bar {
+    padding: 0;
```

Comment:
Also needs:
```
border-radius: 4px;
```

Reply:
This styling should really NOT be in the application; this is supposed to be a re-use of existing style classes. As it is now, this looks completely broken in the elementary stylesheet, and these are workarounds for an incorrect widget/class structure. The borders should go in the Pop! stylesheet, since they might not make any sense in other stylesheets.